source/scripts/Background/migration/v3_3.ts

```diff
@@ -0,0 +1,41 @@
+import { browser } from 'webextension-polyfill-ts';
```

Comment:
It looks like we will need to run through the upgrade path to make sure that the state shape migrated properly.

Reply:
Yeah, agree. It's just adding a new key so pretty low risk but still need to test it.